airflow/example_dags/example_twitter.md

```diff
@@ -0,0 +1,52 @@
+# Example Twitter DAG
+
+<p><strong>Introduction: </strong>This example dag depicts a typical ETL process and is a perfect use case automation scenario for Airflow. <i>Please note that the main scripts associated with the tasks are returning None.</i> The purpose of this DAG is to demonstrate how to write a functional DAG within Airflow.</p>
```

Comment:
I'm surprised to see raw html in markdown, strong would usually be something like `**this is bold**`